README.md

```diff
@@ -0,0 +1,84 @@
+# CNAB to OCI
+
+The intent of CNAB to OCI is to provide a translation tool in between the two formats 
```

Comment:
Missing a final point.

Reply:
The intent is not to provide a tool, which is only for demonstration, but to propose a library for sharing a CNAB bundle on a OCI or Docker registry.

Reply:
Yep